### ISSUE TITLE
Isn't there an option to define input source for recording?

### ISSUE THREAD

**Issue by ManiFaridi** [spatialaudio/python-sounddevice#324]

**Comment by HaHeho** [comment]
One can query the available input and output devices with `print(sd.query_devices())`. If one uses more elaborate hardware one can also specify specific channel(s), e.g. https://github.com/spatialaudio/python-sounddevice/blob/d353fbf6c751a7a91169c614df8f905f2c55821c/examples/rec_unlimited.py#L72-L75

The rest of the that (and other) provided example application shows how it can be conveniently configured.

**Comment by ManiFaridi** [comment]
Sorry, I forgot to mention. I meant an internal source, such as a game sound, windows sound, and so on. How do you configure that?

**Comment by HaHeho** [comment]
See https://github.com/spatialaudio/python-sounddevice/issues/287.

TLDR: You will need the help of an additional virtual sound card, depending on your OS.

**Comment by ManiFaridi** [comment]
OK, thanks.
